cylc/flow/broadcast_mgr.py

```diff
@@ -156,28 +161,21 @@ def expire_broadcast(self, cutoff=None, **kwargs):
             return (None, {"expire": [cutoff]})
         return self.clear_broadcast(point_strings=point_strings, **kwargs)
 
-    def get_broadcast(self, task_id=None):
+    def get_broadcast(self, tokens: 'Optional[Tokens]' = None) -> dict:
         """Retrieve all broadcast variables that target a given task ID."""
-        if task_id == "None":
-            task_id = None
-        if not task_id:
+        if tokens is None or tokens == 'None':
             # all broadcasts requested
             return self.broadcasts
-        try:
-            tokens = Tokens(task_id, relative=True)
-            name = tokens['task']
-            point_string = tokens['cycle']
-        except ValueError:
-            raise Exception("Can't split task_id %s" % task_id)
```

Comment:
I've removed this because tokens objects are pre-parsed so this error cannot occur here any more.